be/src/exec/parquet/encoding_dict.h

```diff
@@ -97,6 +97,7 @@ class DictDecoder final : public Decoder {
     }
 
     Status next_batch(size_t count, ColumnContentType content_type, vectorized::Column* dst) override {
+        _indexes.reserve(count);
```

Comment:
_indexes has already resizd in set_dict(), why reserve here

Reply:
`count` may be greater than `config::vector_chunk_size`, that why some UT failed

Reply:
In actual use, it is impossible to be less than

Reply:
no need to add this reserve just for single test

Reply:
> In actual use, it is impossible to be less than
There is no actual code to enforce that, right?

Reply:
> no need to add this reserve just for single test

That's for preventing future potential bugs, not just for this single test

Reply:
https://github.com/StarRocks/starrocks/blob/0ab1a342f4d25c6212e0de2eadc7360c72aeeca7/be/src/exec/parquet/file_reader.cpp#L431

Reply:
The chunk generated by the storage engine should not be greater than chunk size

Reply:
This is file_reader, not decoder

Reply:
file_reader -> group_reader -> page_reader -> decoder all use the  "n"